classes/genre.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+
+class Genre
+  attr_accessor :name, :items
+
+  def initialize(name)
+    @name = name
+    @items = []
```

Comment:
Maybe it is worth to add @id to this list of variables:
`@id = Random.rand(1...1000)`

I'm asking this because I saw in UML diagram that Genre class has its own @id variable:
![image](https://github.com/Zilola-Nazarova/ruby-capstone-project/assets/61951420/dcf63b40-ce4f-4a22-af83-9bca42f018bc)